### ISSUE TITLE
Error running examples/detector/conf.py

### ISSUE THREAD

**Issue by FilipeMaia** [SPIhub/hummingbird#56]
```shell 
$ ./hummingbird.py -b examples/detector/conf.py
Traceback (most recent call last):
  File "./hummingbird.py", line 67, in <module>
    main()
  File "./hummingbird.py", line 47, in main
    worker = Worker(args.backend)
  File "/Users/filipe/src/hummingbird/src/backend/worker.py", line 39, in __init__
    self.load_conf()
  File "/Users/filipe/src/hummingbird/src/backend/worker.py", line 68, in load_conf
    Worker.conf = imp.load_source('backend_conf', self._config_file)
  File "examples/detector/conf.py", line 7, in <module>
    sim = simulation.simple.Simulation("examples/detector/virus.conf")
  File "/Users/filipe/src/hummingbird/src/simulation/simple.py", line 6, in __init__
    self.input = condor.Input(conf)
AttributeError: 'module' object has no attribute 'Input'
```

**Comment by FilipeMaia** [comment]
Same error running examples/simulation/conf.py

**Comment by FilipeMaia** [comment]
And examples/hitfinding/conf.py

**Comment by mhantke** [comment]
The condor API has changed - I'll fix the examples.

**Comment by FilipeMaia** [comment]
No need, i'm working on it.

**Comment by mhantke** [comment]
Sorry - fixed with 09382129120ede5fd1c2a36827349244458011cb, 37bae3bcad114a50198a814fdeb0ae0029923242, bb1f027913169ce0dc7a89819397e7e8666bf089.
